src/backend/distributed/utils/statistics_collection.c

```diff
@@ -0,0 +1,157 @@
+/*-------------------------------------------------------------------------
+ *
+ * statistics_collection.c
+ *	  Anonymous reports and statistics collection.
+ *
+ * Copyright (c) 2017, Citus Data, Inc.
+ *
+ *-------------------------------------------------------------------------
+ */
+
+#if HAVE_LIBCURL
+
+#include "postgres.h"
+
+#include <curl/curl.h>
+#include <sys/utsname.h>
+
+#include "access/xact.h"
+#include "citus_version.h"
+#include "distributed/metadata_cache.h"
+#include "distributed/statistics_collection.h"
+#include "distributed/worker_manager.h"
+#include "lib/stringinfo.h"
+
+bool EnableStatisticsCollection = true; /* send basic usage statistics to Citus */
+
+static uint64_t NextPow2(uint64_t n);
```

Comment:
Why use `uint64_t` rather than PostgreSQL's types? We don't (AFAIK) use C99 types, instead using the PostgreSQL wrappers for them (`uint64`, `int64`, etc.)